src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -59,10 +59,6 @@ public function getMarking($subject)
     {
         $marking = $this->markingStore->getMarking($subject);
 
-        if (!$marking instanceof Marking) {
```

Comment:
Actually there are a lot of places in the Symfony codebase where we skip checks like this because the interface define the expected type of argument, @fabpot can you argue over this?